pkg/clusteragent/admission/validate/kubernetesadmissionevents/kubernetesadmissionevents_test.go

```diff
@@ -40,6 +41,14 @@ const (
 	eventType      = "kubernetes_admission_events"
 )
 
+// compareText compares text while ignoring the timestamp
+func compareText(expected, actual event.Event) bool {
+	RE := regexp.MustCompile(`\*\*Time:\*\*.*?(\\n|$)`)
+	expectedText := RE.ReplaceAllString(expected.Text, "**Time:** <TIME>\n")
+	actualText := RE.ReplaceAllString(actual.Text, "**Time:** <TIME>\n")
```

Comment:
nit: This variable name is not ideal in idiomatic Go style. I would recommend the following:

```suggestion
	re := regexp.MustCompile(`\*\*Time:\*\*.*?(\\n|$)`)
	expectedText := re.ReplaceAllString(expected.Text, "**Time:** <TIME>\n")
	actualText := re.ReplaceAllString(actual.Text, "**Time:** <TIME>\n")
```

You can find more details about Go coding style here: 
* https://go.dev/doc/effective_go
* https://go.dev/wiki/CodeReviewComments#variable-names
* * https://google.github.io/styleguide/go/decisions#variable-names